kuma/api/v1/plus.py

```diff
@@ -62,3 +64,17 @@ def landing_page_survey(request):
         context["csrfmiddlewaretoken"] = get_token(request)
 
     return JsonResponse(context)
+
+
+@ratelimit(key="user_or_ip", rate="100/m", block=True)
+def landing_page_variant(request):
+    variants = settings.PLUS_PRICE_VARIANTS
+    assert isinstance(variants, list)
+    session_key = "plus-landing-page-variant-v1"
+    variant = request.session.get(session_key)
```

Comment:
I'm confused. If we're getting the variant from the session (via a session cookie) here, why not also get it from the session in the `landing_page_survey` view above, because on the client side won't you always do a GET from this view (to get the price and the variant) **before** you do a GET from the `landing_page_survey` view?

Reply:
Excellent point! I didn't really mean to implement it like this. It used to be that the landing page's React code would conjure up the variant. Then that number would be passed to the `<LandingPageSurvey/>` component that just sends it along. 
But now that the variant decision is in Kuma you have no business submitting the survey at all if you haven't already run through the `landing_page_variant` view function!
So we can simply stop *sending* the variant in the XHR for the survey. And inside `landing_page_survey` we can get the `variant`, not from `request.GET` but from `request.session`. I'll implement that instead.